Source/Charts/Data/Implementations/Standard/PieChartDataEntry.swift

```diff
@@ -23,67 +23,71 @@ open class PieChartDataEntry: ChartDataEntry
     /// - parameter label: The label for the x-axis
     @objc public convenience init(value: Double, label: String?)
     {
-        self.init(value: value, label: label, icon: nil, data: nil)
+        self.init(value: value, label: label, icon: nil, data: nil, imageView: nil)
     }
     
     /// - parameter value: The value on the y-axis
     /// - parameter label: The label for the x-axis
     /// - parameter data: Spot for additional data this Entry represents
     @objc public convenience init(value: Double, label: String?, data: AnyObject?)
     {
-        self.init(value: value, label: label, icon: nil, data: data)
+        self.init(value: value, label: label, icon: nil, data: data, imageView: nil)
     }
     
     /// - parameter value: The value on the y-axis
     /// - parameter label: The label for the x-axis
     /// - parameter icon: icon image
     @objc public convenience init(value: Double, label: String?, icon: NSUIImage?)
     {
-        self.init(value: value, label: label, icon: icon, data: nil)
+        self.init(value: value, label: label, icon: icon, data: nil, imageView: nil)
     }
     
     /// - parameter value: The value on the y-axis
     /// - parameter label: The label for the x-axis
     /// - parameter icon: icon image
     /// - parameter data: Spot for additional data this Entry represents
-    @objc public init(value: Double, label: String?, icon: NSUIImage?, data: AnyObject?)
+    @objc public init(value: Double, label: String?, icon: NSUIImage?, data: AnyObject?, imageView: UIImageView?)
```

Comment:
1. The entry should contain an image, not a view.
2. `UIImageView` is not available on macOS.
3. the image is not necessary to initialize the object.
4. Why can't we use the icon to achieve this?